server/admin.js

```diff
@@ -7,29 +7,27 @@ const { parse, stringify } = require('envfile');
 const pathToenvFile = 'config.env';
 const saltRounds = 13;
 
-if ( !process.env.ADMIN_PW ){
-	throw 'ADMIN_PW must be provided in config.env';
-}
-
 if ( !process.env.PASSWORD_IS_HASHED || process.env.PASSWORD_IS_HASHED !== 'true' ){
-	bcrypt.hash(process.env.ADMIN_PW, saltRounds, function(err, hash) {
-		fs.readFile(pathToenvFile, {encoding:'utf8', flag:'r'}, function(err, data) {
-			if(err) {
-				console.log(err);
-			}
-			let result = parse(data);
-			result['ADMIN_PW'] = hash;
-			result['PASSWORD_IS_HASHED'] = 'true';
-			process.env['ADMIN_PW'] = hash;
-			process.env['PASSWORD_IS_HASHED'] = 'true';	
-
-			fs.writeFile(pathToenvFile, stringify(result), function (err) {
-				if (err) {
+	if ( process.env.ADMIN_PW ){
```

Comment:
May be added to previous if statement for simplicity I just thought that the line might become too long